lib/core/terraform_config/policy.rb

```diff
@@ -0,0 +1,147 @@
+# frozen_string_literal: true
+
+module TerraformConfig
+  # rubocop:disable Metrics/ClassLength
+  class Policy < Base
+    TARGET_KINDS = %w[
+      agent auditctx cloudaccount domain group gvc identity image ipset kubernetes location
+      org policy quota secret serviceaccount task user volumeset workload
+    ].freeze
+
+    GVC_REQUIRED_TARGET_KINDS = %w[identity workload volumeset].freeze
+
+    attr_reader :name, :description, :tags, :target_kind, :gvc, :target, :target_links, :target_query, :bindings
+
+    # rubocop:disable Metrics/ParameterLists, Metrics/MethodLength
```

Comment:
Nit: This can be moved to the end of line 26. That way we don't have to enable the rule again later, it gets disabled only for the block.